docs/reference/cli.md

```diff
@@ -1837,6 +1838,81 @@ interpreter. Use <code>--universal</code> to display the tree for all platforms,
 <p>You can configure fine-grained logging using the <code>RUST_LOG</code> environment variable. (<a href="https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html#directives">https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html#directives</a>)</p>
 </dd></dl>
 
+## uv format
+
+Format Python code in the project.
+
+Formats Python code using the Ruff formatter. By default, all Python files in the project are formatted. This command has the same behavior as running `ruff format` in the project root.
+
+To check if files are formatted without modifying them, use `--check`. To see a diff of formatting changes, use `--diff`.
+
+By default, Additional arguments can be passed to Ruff after `--`.
```

Comment:
Did you mean to write something else here? Maybe the "By default," part should be removed.